osu.Game/Database/RealmArchiveModelImporter.cs

```diff
@@ -286,8 +291,6 @@ public async Task<ExternalEditOperation<TModel>> BeginExternalEditing(TModel mod
         /// <param name="cancellationToken">An optional cancellation token.</param>
         public virtual Live<TModel>? ImportModel(TModel item, ArchiveReader? archive = null, ImportParameters parameters = default, CancellationToken cancellationToken = default) => Realm.Run(realm =>
         {
-            pauseIfNecessary(parameters, cancellationToken);
```

Comment:
I suppose my only concern here is that if someone queues up a bunch of beatmaps to be downloaded via beatmap listing, then they'll import during gameplay, right?

Reply:
Yeah okay I misread. There's so many overloads of `Import()` it's hard to tell which one's used at any one time. This seems fine.